ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/builder/FuluBuilder.java

```diff
@@ -180,6 +191,22 @@ public void validate() {
     validateConstants();
   }
 
+  public void validateBlobSchedule(
```

Comment:
Maybe `validateBlobSchedule(
      final Optional<BlobSchedule> schedule)` and call it the same number of times how many schedules we have?

Reply:
we do call it once... but also we may not be able to have the schedule for a given fork because the variables will disappear. this is why its been done in this way, for compatibility.

Reply:
Ok, now I got it, looks good